styles.css

```diff
@@ -182,22 +195,15 @@ main {
   background-color: #def5f9;
 }
 
-.card-div {
-  margin-left: 10px;
-  margin-right: 10px;
-  justify-content: flex-start;
+.photo {
+  background-size: contain;
 }
 
-.card-div-grey {
-  padding-top: 10px;
-  padding-bottom: 10px;
-  margin-left: 10px;
-  margin-right: 10px;
-  justify-content: flex-start;
-  overflow-wrap:break-word;
-  hyphens:auto;
-  background: lightgray;
-}
+/***************************************************************
+
+NEED TO CREATE MEDIA QUERIES FOR DIFFERENT SCREEN SIZES
+
+***************************************************************/
 
 /* 
 @media screen and (max-width: 480px) {
```

Comment:
It might be easier to use either min or max  with all media queries instead of mixing them. 

Otherwise all this code looks good and well laid out! I'm curious, did you do the initial design on a large screen or as if it were a mobile screen?

Reply:
I started off as a desktop, but I wish I would not have. I'm still pretty shakey with CSS so I was intimidated by starting with mobile-first.